src/cmd/flux-account-service.py

```diff
@@ -659,11 +679,11 @@ def list_queues(self, handle, watcher, msg, arg):
 
             handle.respond(msg, payload)
         except KeyError as exc:
-            handle.respond_error(msg, 0, f"missing key in payload: {exc}")
+            handle.respond_error(msg, 0, f"list-queues: missing key in payload: {exc}")
         except ValueError as exc:
-            handle.respond_error(msg, 0, f"{exc}")
+            handle.respond_error(msg, 0, f"list-queues: {exc}")
         except sqlite3.Error as exc:
-            handle.respond_error(msg, 0, f"a SQLite error occurred: {exc}")
+            handle.respond_error(msg, 0, f"list-queues: a SQLite error occurred: {exc}")
         except Exception as exc:
             handle.respond_error(
                 msg, 0, f"a non-OSError exception was caught: {str(exc)}"
```

Comment:
Instead of so many different types of exceptions, since the handling all seems to be pretty similar, I wonder if you might do something like:

```python
except Exception as exc:
    handle.respond_error(msg, 0, f"list-queues: {type(exc).__name__}: {exc}
```

Reply:
I like this suggestion a lot. Perhaps I should refactor these `try/except` blocks to follow this format instead?